### PR TITLE
[research] Prefer center-1 projection

### PR DESCRIPTION
**Pull request by gonzalobenegas**
Record center-1-bp projection as the default for new multispecies training datasets while preserving existing full-window rules and artifacts as historical controls.
Across five audited regions, center-1 recovered 82.28% of requested species–anchor pairs versus 79.60% for full-window projection.
After correcting the reused CDS checkpoint’s legacy-RoPE loading path, one-seed development AUPRC trajectories show no consistent policy advantage across the region-relevant CDS and enhancer benchmarks.
Experiment #473 held anchors fixed, so anchor-selection policy remains open.

A valid cross-policy validation-loss comparison is unavailable because native validation rows differ by policy and the earlier shared-row replay used an incompatible legacy-RoPE loading path.
The experiment page records that limitation before the QC and AUPRC line trajectories.
The terminal enhancer Mendelian comparison favored full-window projection by 0.056 with a paired interval excluding zero, but distal Complex-trait AUPRC was nearly identical and no additional seed tested replication.

All evaluation evidence uses the public development split containing odd-numbered autosomes and chromosome X.
Held-out even-autosome and chromosome-Y labels remain untouched.

Part of #473.

### CODE REVIEW

**Comment by gonzalobenegas** [comment]
🤖 Addressed the independent review in `f9ad0caa`.

- The recommendation is now explicitly one-seed, development-only, exploratory, and specific to the tested training recipe; the 8/8 additional-seed gate is stated in Findings.
- Complex and SGE comparisons are described as point estimates with official SEs. The page makes no paired-difference or equivalence claim for those endpoints.
- A combined four-panel AUPRC lead figure now shows the opposing CDS and enhancer trajectories on a shared scale. The two redundant single-region AUPRC figures were removed.

Scoped pre-commit checks passed. The independent reviewer rechecked the updated prose and SVG and found all three findings resolved.

**Comment by gonzalobenegas** [comment]
🤖 Updated the evidence figures and page order in fef01212 (rebased onto current main).\n\n- Projection QC now appears first, with exact policy recovery and sampled aligned-coverage values.\n- Shared-row validation loss appears before every VEP result.\n- All model figures show actual full-window and center-1 trajectories in consistent colors; the delta plots were removed.\n- Mendelian AUPRC and Group SMD now share one consistent 2×4 figure covering every region-relevant subset.\n- Complex-trait and SGE trajectories are included with their official uncertainty, and long methodological notes live in captions rather than inside plots.\n- The top-of-page lead figure was removed so figures follow the narrative reading order.\n\nI visually inspected all five rendered figures. The scoped local checks and all GitHub checks pass on the rebased head.

**Comment by gonzalobenegas** [comment]
🤖 @codex review

Please review the current rebased diff at `eab54c744b2e6be86dbfb8226a140150e5993332`.
The earlier review covered superseded pre-repair evidence; this revision replaces that interpretation with the corrected canonical CDS trajectories, revised QC, and the accepted center-1 default.

**Comment by gonzalobenegas** [comment]
🤖 @codex review

Please independently review the final published diff at head `78c6c42c`, including the QC-first reading order, the corrected actual-value CDS and enhancer AUPRC line plots, the one-seed interpretation, and the stated enhancer caveat.

**Comment by gonzalobenegas** [comment]
🤖 Final knowledge-base interpretation is ready for human review at `78c6c42c`.

- Projection QC appears first and reports exact recovery plus paired external-flank diagnostics; overlapping bar-top labels are removed.
- The repaired CDS and unaffected enhancer comparisons use actual-value AUPRC line trajectories across all nine checkpoints, with region-relevant subsets only, per-checkpoint ±1 SE, and prevalence baselines.
- The page explicitly withdraws the invalid shared-row loss comparison and explains why no valid cross-policy validation-loss figure remains.
- The conclusion selects center-1 as the simpler default for new projection datasets while preserving full-window rules and artifacts and avoiding an equivalence claim.
- The terminal enhancer Mendelian result that favors full-window remains explicit, together with the missing replication caveat.
- Held-out labeled VEP data remain untouched.

All final-head CI checks pass, the branch is mergeable, and the independent Codex review found no major issues on exact head `78c6c42c`.

File: docs/research/questions/genomic-anchors.md [hunk 1]
```diff
@@ -1,31 +1,34 @@
 # How should genomic anchors be selected and projected across species?
 
 > [!NOTE]
-> **TL;DR:** The current full-window projection contract supports viable training datasets, but its anchor criterion, fragment semantics, and homology-versus-yield tradeoff remain unsettled; confidence is high in the baseline behavior and low in its optimality, with a matched downstream comparison still needed.
+> **TL;DR:** Center-1-bp projection is the default for new multispecies training datasets because it is simpler than full-window projection, increased aggregate species–anchor recovery, and produced broadly similar one-seed development AUPRC trajectories for CDS and enhancer specialists; existing full-window artifacts remain valid historical controls.
```
Comment (gonzalobenegas) [inline docs/research/questions/genomic-anchors.md]: we only got insights about projection, but none about anchor selection, that's very much an open question. you shouldn't erase it

File: docs/research/experiments/473-center-seeded-projection.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+The terminal Mendelian distal AUPRC favored full-window projection by 0.0557, with a paired 95% interval of [-0.100, -0.016] for center-1 minus full-window.
+The Complex-trait distal difference was +0.0015 for center-1, and no additional seed tested whether the Mendelian result replicates.
+
+## Limitations
+
+- One training seed cannot establish statistical equivalence or the reproducibility of endpoint-level differences.
+- The arms were matched on presented sequences and tokens, not effective epochs, because the policies produced different row counts.
+- Recovery and sampled external-flank measurements do not establish locus-level homology.
+- CDS and enhancer-centered cCREs have matched downstream training comparisons; three other regions have projection QC only.
+- Held-out even-autosome and chromosome-Y VEP data remain untouched, so the interpretation is development-only.
```
Comment (gonzalobenegas) [inline docs/research/experiments/473-center-seeded-projection.md]: remove this line. this is not a limitation

File: docs/research/questions/genomic-anchors.md [hunk 3]
```diff
@@ -78,13 +60,15 @@ Projection yield alone is insufficient.
...
+- [Experiment #473](../experiments/473-center-seeded-projection.md) compared full-window and center-1 projection on fixed anchors with recovery, reverse-trace QC, matched-token training, paired Mendelian uncertainty, Complex traits, and SGE.
+  Its corrected one-seed development trajectories are broadly similar, supporting center-1 as the simpler default while retaining full-window artifacts as historical controls.
 
 </details>
 
 <details>
 <summary>Possible directions</summary>
 
-- Which projection semantics best balance species recovery with confidence that the extracted target window is homologous to the human anchor: full-window projection, center-seeded projection, multiple projected landmarks, or fragment/locus-based alternatives?
+- For regions beyond CDS and enhancer-centered cCREs, which projection semantics best balance species recovery with confidence that the extracted target window is homologous to the human anchor: full-window projection, center-seeded projection, multiple projected landmarks, or fragment/locus-based alternatives?
```
Comment (gonzalobenegas) [inline docs/research/questions/genomic-anchors.md]: I think you can remove this, not a pressing question

File: docs/research/experiments/473-center-seeded-projection.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+| CDS | +0.00040 | [-0.00312, 0.00393] | 214 | -0.163 bp | [-1.617, 1.010] | 209 |
+| ncRNA exon | +0.00484 | [-0.00231, 0.01200] | 51 | -0.400 bp | [-3.952, 2.750] | 40 |
+| TSS / 5′ UTR | +0.00580 | [-0.01104, 0.02264] | 46 | +0.889 bp | [-2.938, 4.245] | 45 |
+| 3′ UTR | +0.00080 | [-0.01044, 0.01205] | 49 | +1.143 bp | [0.387, 2.204] | 49 |
+
+_Paired center-1 minus full-window mean aligned-to-anchor fraction and total external-flank bases among the sampled traces._
+
+Two examples make the policy difference concrete:
+
+- CDS anchor `win_chr10_000116477` in *Dinomys branickii* maps both policies to `DinBra_scaffold_14185` on the plus strand. Full window uses four fragments and emits `[89487, 89742)`; center-1 uses one fragment and emits `[89485, 89740)`. Both retain 249 aligned anchor bases and three internal unaligned bases.
```
[the remark this answers is not in the document]
Comment (gonzalobenegas) [reply docs/research/experiments/473-center-seeded-projection.md]: 🤖 Addressed in the current page. Captions and limitation bullets now use one prose sentence per source line, and the sentence-boundary scan found no remaining violations.

File: docs/research/questions/genomic-anchors.md (hunk 1 above)
Comment (gonzalobenegas) [reply docs/research/questions/genomic-anchors.md]: 🤖 Addressed in 332d4d74. The TL;DR now says anchor selection remains open because experiment #473 held anchors fixed, and the current answer states that the experiment does not resolve anchor-selection policy.

File: docs/research/experiments/473-center-seeded-projection.md (hunk 2 above)
Comment (gonzalobenegas) [reply docs/research/experiments/473-center-seeded-projection.md]: 🤖 Removed this line from Limitations in 332d4d74.

File: docs/research/questions/genomic-anchors.md (hunk 3 above)
Comment (gonzalobenegas) [reply docs/research/questions/genomic-anchors.md]: 🤖 Removed this direction in 332d4d74.
